tests/test_sv.py

```diff
@@ -117,8 +120,28 @@ def test_validation_no_errors(self):
                 "CONNECTION_TYPE": ["3to5", "5to5"],
                 "DNA_SUPPORT": ["Yes", "No"],
                 "RNA_Support": ["Yes", "No"],
+                "SITE1_CHROMOSOME": [1, 22],
+                "SITE2_CHROMOSOME": ["X", "2"],
             }
         )
         error, warning = self.sv_cls._validate(sv_df)
         assert error == ""
         assert warning == ""
+
+    def test_validation__validate_chromosome_is_called(self):
+        """Tests that _validate_chromosome is called twice
+        for the two chromosome columns that sv files may have"""
+        sv_df = pd.DataFrame(
+            {
+                "sample_id": ["GENIE-SAGE-ID1-1", "GENIE-SAGE-ID2-1"],
+                "SITE1_CHROMOSOME": [1, 22],
+            }
+        )
+        with mock.patch(
+            "genie.validate._validate_chromosome", return_value=("", "")
+        ) as validation__validate_chromosome_mock:
+            self.sv_cls._validate(sv_df)
+            assert validation__validate_chromosome_mock.call_count == 2, (
```

Comment:
Nit: This is fine for this purpose since we have tested this function.  BUT, for future reference, here's how I would tackle this to save time.

* Reduce the dataframe to length of one so the function is called only once.
* Do something like this to capture the[ actual calls](https://github.com/Sage-Bionetworks/Genie/blob/f6cea5b0794506549ff6cf7ce74290628ffc037b/tests/test_database_to_staging.py#L49-L58). (This is potentially over testing)